gurla/express-hello-world
# Add AI News endpoint

This PR adds a new `/api/ai-news` endpoint that returns the latest AI news headlines and summaries. Changes include:

1. Added new endpoint `/api/ai-news` that returns 5 recent AI news stories
2. Updated README.md with documentation for the new endpoint
3. Each news item includes:
   - Title
   - Summary
   - Source
   - Date

The news data is currently static but could be updated to fetch from real news sources in the future.

To test:
1. Start the server
2. Access http://localhost:3000/api/ai-news
3. Verify the response format matches the documentation
